apps/web/src/styles/accessibility.css

```
@@ -0,0 +1,387 @@
+  padding: inherit;
+  margin: inherit;
+  overflow: visible;
+  clip: auto;
+  white-space: normal;
+}
+
+/**
+ * Hide from screen readers (decorative elements)
+ */
+.aria-hidden {
+  aria-hidden: true;
```

Biome (2.3.13)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)